lib/src/screens/wallet_list/filtered_list.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_mobx/flutter_mobx.dart';
+import 'package:mobx/mobx.dart';
+
+class FilteredList extends StatefulWidget {
+  FilteredList({
+    required this.list,
+    required this.itemBuilder,
+    required this.updateFunction,
+  });
+
+  final ObservableList<dynamic> list;
+  final Widget Function(BuildContext, int) itemBuilder;
+  final Function updateFunction;
+
+  @override
+  FilteredListState createState() => FilteredListState();
+}
+
+class FilteredListState extends State<FilteredList> {
+  @override
+  Widget build(BuildContext context) {
+    return Observer(
+      builder: (_) => ReorderableListView.builder(
+        physics: const BouncingScrollPhysics(),
+        itemBuilder: widget.itemBuilder,
+        itemCount: widget.list.length,
+        onReorder: (int oldIndex, int newIndex) {
+          setState(() {
```

Comment:
wouldn't this cause a duplicate rebuild? wouldn't the observer already rebuild it when the observable list changes?